cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -190,6 +191,62 @@ def test_schedulable_memory_validator(schedulable_memory, ec2memory, instance_ty
     assert_failure_messages(actual_failures, expected_message)
 
 
+# ---------------- Flexible Instance Types validators ---------------- #
+
+
+@pytest.mark.parametrize(
+    "queue_name, compute_resource_name, instance_types_info, disable_simultaneous_multithreading, efa_enabled, "
+    "placement_group_enabled, expected_message",
+    [
```

Comment:
add a test case where there is no failure, for both disable_simultaneous_multithreading choices